lib/checks/aria/aria-busy.json

```diff
@@ -1,6 +1,7 @@
 {
   "id": "aria-busy",
   "evaluate": "aria-busy-evaluate",
+  "deprecated": true,
```

Comment:
Deprecation should probably go into its own PR so it's a separate line in the changelog.